### PR TITLE
GHA for generating formula

### PR DESCRIPTION
**Pull request by kwigley**
this PR revamps the ability to generate formula via GHA

### CODE REVIEW

**Comment by leahwicz** [comment]
@kwigley could you give a brief summary of the TODOs or issues left to work on here please? I was looking to pick this up sooner than later but wanted to make sure I knew what to look at closer

**Comment by kwigley** [comment]
this is ready to roll

File: .github/formula-template.j2 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  desc "{{ summary|capitalize }}"
+  homepage "{{ homepage }}"
+  url "{{ source_url }}"
+  sha256 "{{ source_checksum }}"
+  # license ""
+  revision 1
+
+  depends_on "rust" => :build
+{%- if package == 'dbt-postgres' %}
```
Comment (leahwicz) [inline .github/formula-template.j2:12]: I _think_ Redshift needs the postgres dependency as well

Comment (kwigley) [reply .github/formula-template.j2]: fix and tested locally 👍

**Review by leahwicz — APPROVED** [review]

**Pull request merged**